app/src/main/java/org/mozilla/fenix/components/toolbar/TabCounterToolbarButton.kt

```diff
@@ -77,4 +78,8 @@ class TabCounterToolbarButton(
     private fun updateCount(count: Int) {
         reference.get()?.setCountWithAnimation(count)
     }
+
+    private fun isPrivate(store: BrowserStore): Boolean {
```

Comment:
We're currently working on uplifting this class to A-C, so let's add this + tests there as well: https://github.com/mozilla-mobile/android-components/pull/8872

Reply:
We could also make this a `BrowserStore` ext if we end up fixing different versions of this bug